spec/models/searchgov_url_spec.rb

```diff
@@ -29,42 +27,47 @@
     describe '.fetch_required' do
 
       it 'includes urls that have never been crawled and outdated urls' do
-        expect(SearchgovUrl.fetch_required.pluck(:url)).
+        expect(described_class.fetch_required.pluck(:url)).
           to include('http://www.agency.gov/new', 'http://www.agency.gov/outdated')
       end
 
       it 'does not include current, crawled and not enqueued urls' do
-        expect(SearchgovUrl.fetch_required.pluck(:url)).
+        expect(described_class.fetch_required.pluck(:url)).
           not_to include('http://www.agency.gov/current')
       end
 
       it 'includes urls that have been enqueued for reindexing' do
-        expect(SearchgovUrl.fetch_required.pluck(:url)).
+        expect(described_class.fetch_required.pluck(:url)).
           to include 'http://www.agency.gov/enqueued'
       end
 
       it 'includes urls last crawled more than 30 days and crawl status is ok' do
-        expect(SearchgovUrl.fetch_required.pluck(:url)).
+        expect(described_class.fetch_required.pluck(:url)).
           to include 'http://www.agency.gov/crawled_more_than_month'
       end
     end
   end
 
   describe 'validations' do
-    it 'requires a valid domain' do
-      searchgov_url = SearchgovUrl.new(url: 'https://foo/bar')
-      expect(searchgov_url).not_to be_valid
-      expect(searchgov_url.errors.messages[:searchgov_domain]).
-        to include 'is invalid'
+    describe 'when the URL domain does not already exist in our database' do
```

Comment:
This should be a `context` rather than a `describe` block

Reply:
Changed.